[16.0][FIX] web: statusbar field on mobile doesn't show current value

SplashS opened the pull request:
Description of the issue/feature this PR addresses:
This PR fixes displaying oа current value for status bar field on mobile.

Current behavior before PR:
There is bug on form view.
F.x. in CRM Lead form current stage isn't shown on mobile. 

Desired behavior after PR is merged:
Current value for status bar is shown on mobile.




---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/101511).

SplashS commented:
@mcm-odoo Hello! Please take a look at this PR.
It touches your code.

SplashS commented:
Before this PR:
<img width="574" alt="image" src="https://user-images.githubusercontent.com/6389955/196821807-8cbbd367-9e54-4cf4-ae9c-3c3b7fafe640.png">
After this PR:
<img width="574" alt="image" src="https://user-images.githubusercontent.com/6389955/196822281-ddd729b7-449c-4999-a181-df529d10f4c8.png">

Iucapad commented:
Looks good to me. Could you please add a test in the _statusbar_field_tests.js_ file?
There is already a similar test for selection field
This for example should do the work for your fix:

`QUnit.test(
        "folded statusbar widget on many2one field has selected value in the toggler",
        async function (assert) {
            registry.category("services").add("ui", {
                start(env) {
                    Object.defineProperty(env, "isSmall", {
                        value: true,
                    });
                    return {
                        bus: new EventBus(),
                        size: 0,
                        isSmall: true,
                    };
                },
            });
            await makeView({
                type: "form",
                resModel: "partner",
                resId: 1,
                serverData,
                arch: `
                <form>
                    <header>
                        <field name="trululu" widget="statusbar" options="{'clickable': 1}" />
                    </header>
                </form>`,
            });

            assert.containsOnce(target, ".o_statusbar_status button.dropdown-toggle:contains(aaa)");
        }
    );`

SplashS commented:
@Iucapad Thank you for your reply and support.
I added test for this fix.
But fortunately I found existing test and just added assert.
New assert fails without my fix, so it is enough for my opinion.

Iucapad commented:
Okay for me then :)
Thanks for the fix!
@robodoo r+
